integration-tests/olminstall/helpers/tekton_util.py

```
@@ -0,0 +1,210 @@
+    run(["git", "config", "--global", f"http.{host_origin}.sslVerify", "false"])
+
+
+def parse_junit_summary(artifacts_dir: str | Path) -> dict[str, int]:
+    """Parse JUnit XML files and return aggregate test counts.
+
+    Returns dict with keys: ``total``, ``passed``, ``failures``, ``errors``, ``skipped``.
+    """
+    total = failures = errors = skipped = 0
+    for xml_path in sorted(Path(artifacts_dir).glob("*.xml")):
+        try:
+            tree = ElementTree.parse(xml_path)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)